Drivers/Inc/Minions.h

```diff
@@ -3,28 +3,32 @@
 #include <stdbool.h>
 #include "BSP_GPIO.h"
 
+// used to index into lookup table
+// if changed, PINS_LOOKARR should be changed in Minions.c
+#define FOREACH_MinionPin(PIN) \
+        PIN(IGN_1)   \
+        PIN(IGN_2)  \
+        PIN(REGEN_SW)   \
+        PIN(FOR_SW)  \
+        PIN(REV_SW)  \
+        PIN(CRUZ_EN)  \
+        PIN(CRUZ_ST)  \
+        PIN(BRAKELIGHT)  \
+
+#define GENERATE_ENUM(ENUM) ENUM,
+#define GENERATE_STRING(STRING) #STRING,
+
+typedef enum MINIONPIN_ENUM {
+    FOREACH_MinionPin(GENERATE_ENUM)
+    NUM_MINIONPINS,
+} MinionPin_t;
+
```

Comment:
same as before. If the only reuse of these foreach macros is in the CLI, I feel like there has to be a more elegant solution to this